templates/single-product/product-attributes.php

```diff
@@ -41,7 +41,25 @@
 		<tr>
 			<th><?php echo wc_attribute_label( $attribute->get_name() ); ?></th>
 			<td><?php
-				$values = $attribute->is_taxonomy() ? wc_get_product_terms( $product->get_id(), $attribute->get_name(), array( 'fields' => 'names' ) ) : $attribute->get_options();
+				$values = array();
+
+				if ( $attribute->is_taxonomy() ) :
```

Comment:
In other templates, when we have a multi-line block of PHP, regardless of this being in a template file we've chosen to use bracket notation. Should probably use that here?